e2e/pages/events/addApplicationDocumentsEvent.page.js

```diff
@@ -34,6 +35,7 @@ module.exports = {
 
   uploadFile(file, index) {
     I.attachFile(this.fields(index).document, file);
+    I.waitForInvisible(this.fields(index).documentUploading, 20);
```

Comment:
there is 2 sec wait period in stepListener.js, hopefully it wont lead to situation where 'Uploading' text disappears before 2 sec elapse